### DISCUSSION TITLE
Further future of the project?

### DISCUSSION THREAD

**Discussion by melishev** [vueuse/vueuse#3963] [Q&A]
What is the status of the project, where is the team? Our PR has been waiting to be merged for a month and nothing happens to it. The same situation with many other PRs that are not related to documentation, only documentation is being merged.

Can you give comments on when we should expect news from you about all PRs?

**Comment by ilyaliao** [comment]
The team has expanded with several new members, and I believe most PRs have been merged into v12. If there are any urgent issues, feel free to remind us using @ or Send a message on a Discord server!

We'll do our best :)
